### DISCUSSION TITLE
rate limit exceeded for plan FREE_EDUCATIONAL (Github Copilot)

### DISCUSSION THREAD

**Discussion by joepalexander** [community/community#43841] [Copilot Conversations]
### Select Topic Area

Bug

### Body

getting the error
`[ERROR] [fetch] [2023-01-10T12:37:48.122Z] Unhandled status from server:,429,{"error":{"message":"rate limit exceeded","internal_message":"rate limit exceeded for plan FREE_EDUCATIONAL","type":"client_error"}}
` when using github copilot on the student plan

**Comment by psapezhka** [comment]
The same for the `FREE_ENGAGED_OSS` plan

**Reply by robert-hoffmann** [reply]
What plan is `free_engaged_oss` supposedly ?

**Comment by TiagoRibeiro25** [comment]
same

**Comment by raulibanez** [comment]
Right

**Comment by Cwarcup** [comment]
Same thing here. The issue usually goes away if I reload the window, but comes back relatively quickly.

Does anybody know how to cancel the extended trial so I can just pay for it? I found [this](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-for-individuals-subscription) but I don't seem to have the "edit" dropdown in my billing & plans section.

I periodically get emails stating my trial is ending on X date followed by another saying it's been renewed. 

<img width="798" alt="Screenshot 2023-01-10 at 11 07 05 AM" src="https://user-images.githubusercontent.com/88569957/211639620-50320692-f0a0-419a-af99-c7f11605509a.png">

**Reply by dylancl** [reply]
Also interested - I would rather pay than be locked into this rate limited Technical Preview.

**Reply by Cwarcup** [reply]
Same here.

**Reply by shivam13juna** [reply]
Joining this chain for updates, same here

**Reply by WillKoehrsen** [reply]
Note to GitHub: make it easy for us to pay you! It's ridiculous there's no option to pay for CoPilot if you have the free version.

**Comment by AsrieltheGoat** [comment]
Same here bro
I still don't know why they did that
Are they following ChatGPT lead?
![image](https://user-images.githubusercontent.com/25924894/211766444-a4bc8051-8eb3-47d3-8248-afe71ceca2f6.png)

**Comment by Veyg** [comment]
I am dealing with the same issue, 
`[INFO] [default] [2023-01-11T09:18:28.168Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-01-11T09:18:28.232Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 64 ms
[ERROR] [fetch] [2023-01-11T09:18:28.233Z] Unhandled status from server:,429,{"error":{"message":"rate limit exceeded","internal_message":"rate limit exceeded for plan FREE_EDUCATIONAL","type":"client_error"}}`

Is this package limited by number of students using it, or is it the policy here?

**Reply by ReaVNaiL** [reply]
Apparently a new policy or something that hasn't been disclosed yet. 

Edit: I think it was answered here #43673 by the support team.

**Comment by raulibanez** [comment]
Original thread here:
https://github.com/orgs/community/discussions/43673#discussioncomment-4656573

**Comment by Akash1134** [comment]
Hi there,

Some update on this issue:

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

**Reply by WillKoehrsen** [reply]
Thank you for the response. How can we cancel a free subscription and get on the paid plan? Currently, there is no way for free users to upgrade to the paid plan.

**Reply by Cwarcup** [reply]
Same question here. I would love to be able to keep my student plan AND pay for Copilot if it provides improved stability.

**Reply by FraserLee** [reply]
@Akash1134 
Same situation here. Because I'm eligible for the student plan, there doesn't seem to be any way to pay for a higher rate limit.

<img width="488" alt="Screen Shot 2023-01-12 at 3 07 39 PM" src="https://user-images.githubusercontent.com/30442265/212170017-8ab4aab5-1251-4e24-8fd7-29129cb83388.png">
<img width="608" alt="Screen Shot 2023-01-12 at 3 11 29 PM" src="https://user-images.githubusercontent.com/30442265/212170707-da0dcbf4-dbe9-4a81-8326-b84386d41516.png">

I would gladly pay for a higher tier if I'm able to. Presently with the rate limit so intense, it's not really usable.

**Reply by baggiponte** [reply]
Hi, still having the same issue
